files/quizgrader/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '2'
```

Comment:
oooh, looks great! I'm going to review the .env when I get home and actually have the key 💃 

1) I suppose you want 'image' instead of 'build', right? I suppose this might work regardless. 
2) Can you expose it on any other port on the host? I'm automatically deploying nginx + letsencrypt to all web servers, so you'll also get https for free :D :D :D 
3) Can you add a healthcheck? The docker deployment script I created relies on it :)
e.g.
```
healthcheck:
      test: ["CMD", "curl", "-f", "http://localhost:8080/openmrs/"]
      timeout: 20s
```

After I recreate the VM, I can give you the URL to use in the webhook on your image in dockerhub (https://hub.docker.com/r/burke/quizgrader/), to deploy automatically when you deploy a new version of your image. 

You can also move that image to openmrs organisation if you want, but that's not mandatory.